src/test/java/org/sqlite/DBMetaDataTest.java

```diff
@@ -1423,12 +1425,12 @@ public void columnOrderOfgetExportedKeys() throws SQLException {
         stat.executeUpdate(
                 "create table address (pid integer, name, foreign key(pid) references person(id))");
 
-        ResultSet exportedKeys = meta.getExportedKeys("default", "global", "person");
+        ResultSet exportedKeys = meta.getExportedKeys("default", null, "person");
         assertThat(exportedKeys.next()).isTrue();
         assertThat(exportedKeys.getString("PKTABLE_CAT")).isEqualTo("default");
-        assertThat(exportedKeys.getString("PKTABLE_SCHEM")).isEqualTo("global");
+        assertThat(exportedKeys.getString("PKTABLE_SCHEM")).isEqualTo(null);
```

Comment:
same here, it should be the actual schema of the table, not the parameter that was passed